### ISSUE TITLE
aoc: unable to download a file at the root of shared folder

### ISSUE THREAD

**Issue by junkimu** [IBM/aspera-cli#157]
Found in 4.18.1 under Windows.

In the following examples, `projX` is a folder shared by an another user which contains one file, `hello.txt`

```
>ascli --version
4.18.1

>ascli aoc files browse /projX
+----------------+----------------------+
| key            | value                |
+----------------+----------------------+
| name           | projX                |
| type           | link                 |
| recursive_size |                      |
| size           |                      |
| modified_time  | 2023-05-08T23:44:57Z |
| access_level   | edit                 |
+----------------+----------------------+

>ascli aoc files browse /projX/
+----------------+--------+----------------+------+----------------------+--------------+
| name           | type   | recursive_size | size | modified_time        | access_level |
+----------------+--------+----------------+------+----------------------+--------------+
| hello.txt      | file   |                | 5    | 2024-09-04T03:57:13Z | edit         |
+----------------+--------+----------------+------+----------------------+--------------+

>ascli aoc files browse /projX/hello.txt
+----------------+----------------------+
| key            | value                |
+----------------+----------------------+
| name           | hello.txt            |
| type           | file                 |
| recursive_size |                      |
| size           | 5                    |
| modified_time  | 2024-09-04T03:57:13Z |
| access_level   | edit                 |
+----------------+----------------------+

>ascli aoc files download --to-folder=%TEMP% /projX/hello.txt
 Time: 00:00:01 =---=---=---=---=---=---=---=---=---=-- 0% 0 Mbps  ETA: ??:??:??
E, [2024-09-04T11:14:03.787983 #8248] ERROR -- : ascp failed (pid 20976 exit 1)
W, [2024-09-04T11:14:03.788382 #8248]  WARN -- : An error occurred during transfer: Server aborted session: Docroot check failed: Docroot is not a directory.
W, [2024-09-04T11:14:03.788588 #8248]  WARN -- : Resuming in 2 seconds (retry left:6)
E, [2024-09-04T11:14:08.587838 #8248] ERROR -- : ascp failed (pid 2680 exit 1)
W, [2024-09-04T11:14:08.588065 #8248]  WARN -- : An error occurred during transfer: Server aborted session: Docroot check failed: Docroot is not a directory.
E, [2024-09-04T11:14:08.588305 #8248] ERROR -- : non-retryable error
ERROR: Transfer: Server aborted session: Docroot check failed: Docroot is not a directory.
```

Same test under 4.11.0 succeeds.
```
>ascli --version
4.11.0

>ascli aoc files browse /projX
+----------------+--------+----------------+------+----------------------+--------------+
| name           | type   | recursive_size | size | modified_time        | access_level |
+----------------+--------+----------------+------+----------------------+--------------+
| hello.txt      | file   |                | 5    | 2024-09-04T03:57:13Z | edit         |
+----------------+--------+----------------+------+----------------------+--------------+

>ascli aoc files browse /projX/
+----------------+--------+----------------+------+----------------------+--------------+
| name           | type   | recursive_size | size | modified_time        | access_level |
+----------------+--------+----------------+------+----------------------+--------------+
| hello.txt      | file   |                | 5    | 2024-09-04T03:57:13Z | edit         |
+----------------+--------+----------------+------+----------------------+--------------+

>ascli aoc files browse /projX/hello.txt
+-----------+------+----------------+------+----------------------+--------------+
| name      | type | recursive_size | size | modified_time        | access_level |
+-----------+------+----------------+------+----------------------+--------------+
| hello.txt | file |                | 5    | 2024-09-04T03:57:13Z | edit         |
+-----------+------+----------------+------+----------------------+--------------+

>ascli aoc files download --to-folder=%TEMP% /projX/hello.txt
hello.txt                                     100%    5               00:00
Completed: 0K bytes transferred in 0 seconds
 (0K bits/sec), in 1 file.
 ```

**Comment by laurent-martin** [comment]
Thanks, I have verified it's a bug.

**Comment by laurent-martin** [comment]
Workaround: 
insert a space just after the / before file name:

```console
> ascli aoc files browse /projX/ hello.txt
```

**Comment by laurent-martin** [comment]
Fix is in beta here:

https://ibm.biz/aspera-cli-beta

**Comment by junkimu** [comment]
I patched [fd39763](https://github.com/IBM/aspera-cli/commit/fd39763263abd495843980d3d2653f7abc313311) into my local installation and verified the download problem is fixed.

**Comment by laurent-martin** [comment]
Thanks, so it will be in next release.
